runtime/src/message_processor.rs

```diff
@@ -218,6 +226,17 @@ impl InvokeContext for ThisInvokeContext {
     fn get_programs(&self) -> &[(Pubkey, ProcessInstruction)] {
         &self.programs
     }
+
+    fn log_enabled(&self) -> bool {
+        log_enabled!(log::Level::Info) || self.log_collector.is_some()
+    }
+
+    fn log(&mut self, message: &str) {
+        info!("{}", message);
```

Comment:
I guess you are sticking the calls to `info!` here, but if no log_collector is provided then no logs are output to Solana logger?

Reply:
yeah

Reply:
Hmm, will make running programs on localnet, in particular a sequence of instructions more difficult to debug.

Reply:
Opps, I read your question wrong.  This line,
https://github.com/solana-labs/solana/blob/752328aa3ad1c20b981603c61cf3c1ee5d2982ca/runtime/src/message_processor.rs#L235
will produce the same logging as before.